mods/d2k/rules/structures.yaml

```diff
@@ -16,8 +16,7 @@
 		RemoveInstead: true
 	Buildable:
 		Queue: Building
-		Prerequisites: construction_yard
-		BuildPaletteOrder: 10
+		BuildPaletteOrder: 10 
```

Comment:
You have messed something up here I guess (otherwise it wouldn't mark this line added). Line ending?